### DISCUSSION TITLE
Execute-ProcessAsUser - General Question

### DISCUSSION THREAD

**Discussion by dpuls** [PSAppDeployToolkit/PSAppDeployToolkit#722] [Q&A] [answered]
Hello Community,

I have a general question about the function "Execute-ProcessAsUser".

The following scenario:
I have created an application with PSADT and would like to distribute it via SCCM. What is special, however, is that the process / the *.exe should be executed on the client in the context of another user who has local user rights.
This is absolutely necessary, otherwise the application cannot do its job.

As far as I understand, this works with the "Execute-ProcessAsUser" function, but how do I ensure that the process is permanently executed as this user?
Is it even possible that I enter another user like "domain\admuser" during the installation or do I have a problem understanding the function?

Thanks
Daniel

**Comment by cyborgrd** [comment]
This function works only with user who is currently logged on. The function creates a scheduled task which runs only one time with user context.

**Comment by That-Annoying-Guy** [comment, marked as answer]
A better solution is to use ActiveSetup. ActiveSetup doesn't need to have the user logged on to work.

FYI: the better place to ask questions is here: https://discourse.psappdeploytoolkit.com/c/thetoolkit/general-discussion
